flax/core/tests/lift_test.py

```diff
@@ -32,12 +32,10 @@ def g(scopes, _):
         scope, a = scopes
         self.assertEqual(a.parent, scope)
       
-      lift.vmap(g, variable_in_axes={}, variable_out_axes={}, split_rngs={})((scope, a), jnp.ones((1,)))
+      lift.vmap(g, variable_axes={}, split_rngs={})((scope, a), jnp.ones((1,)))
```

Comment:
Are there any unit tests for the lifted transformations? Or is that still to be done in a subsequent PR?

Reply:
yes core.lift still needs elaborate testing in a subsequent PR. Although the linen transform tests tough a lot of this stuff as well.